### DISCUSSION TITLE
Assemble several photos into a gif animation

### DISCUSSION THREAD

**Discussion by bartgw** [tannerhelland/PhotoDemon#487] [Q&A]
Hallo,

I want to assemble several images into a gif animation in PhotoDemon. I know that adding photos into a single layer can help to convert all figures into a gif animation, however, I cannot add all images to a single layer at once. Is there a way to assemble more than 50 images into an animation at once?

**Comment by tannerhelland** [comment]
Hello @bartgw.  I apologize that it has taken me so long to reply to your question.

[PhotoDemon nightly builds](https://photodemon.org/download/#nightly) have some new features that make it much easier to create an animation from individual image files.  Here are my suggested steps:

1) Load the first frame of your animation as a new image.
2) Go to `Layer > Add > From file` and select all the remaining frames of your image (using `Shift+Click` or `Ctrl+Click`).  Add all remaining frames at once.
3) (Optionally) If you need to adjust frame order at all, you can Drag+Drop frames into a new order using the Layers toolbox.
4) Go to `Image > Animation` and change `animation enabled for this image` to `YES`.  This exposes new frame time settings and an interactive preview.  Choose your desired settings, then click `OK`.

This produces an animation that can be exported to any animation format you like, including GIF, PNG, WebP, and others.
